media-sound/wavplay/wavplay-2.0.ebuild

```diff
@@ -12,9 +12,8 @@ SRC_URI="https://downloads.sourceforge.net/${PN}/${P}.tar.gz"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 x86"
-IUSE=""
 
-PATCHES=( "${FILESDIR}"/${PN}-2.0-pkgdata-install-fix.patch )
```

Comment:
And not an "issue".

Reply:
But it's not an improvement? :) why do you need to make this change at all?

Reply:
Please don't make that change at all, at least for my packages. I consider it stylistically correct to do the quotes earlier, rather than later. Thanks!

(This isn't my package, it's just that I saw this PR and the topic came up from someone else already.)

But in general, style changes should probably NOT be done to packages that are not yours, unless there is a tree policy for it (such as the use of tab indentation over 4-space indentation.)

Reply:
https://devmanual.gentoo.org/ebuild-writing/file-format/#indenting-and-whitespace
https://devmanual.gentoo.org/ebuild-writing/common-mistakes/#wrongfully-used-spaces-instead-of-tabs